programmers-coding-test/minjae/dfs-bfs/유기농배추.js

```diff
@@ -0,0 +1,66 @@
+const readline = require("readline"); 
+const rl = readline.createInterface({ 
+    input: process.stdin, 
+    output: process.stdout, 
+}); 
+
+let input = [];
+let farm = [];
+let testCase = 0;
+let farmHeight, farmWidth, cabbages;
+formHeight = formWidth = cabbages = 0;
```

Comment:
여러 변수에 값 대입할 때는 그냥 이걸로 통일시켜주는게 좋을 것 같습니다!
```suggestion
let [farmHeight, farmWidth, cabbages] = [0, 0, 0];
```